cppForSwig/cryptopp/configure.ac

```diff
@@ -21,12 +21,9 @@ AC_ARG_ENABLE([debug],
               [want_debug="$enableval"], [want_debug=no])
 
 if test "x$want_debug" = "xyes" -a $ac_cv_c_compiler_gnu != no; then
-  CFLAGS="$CFLAGS -O0 -g"
-  CXXFLAGS="$CXXFLAGS -O0 -g"
+  CFLAGS="-O0 -g"
+  CXXFLAGS="-O0 -g"
   AC_DEFINE([DEBUG], 1, [Define for debugging])
-else
```

Comment:
C/C++ flags are assigned for debug but not for release builds. At the same time if the flags are default for a release build (-02 -g), this means the cflags will result as being empty. I do manually change the upper cflags (in the root configure.ac) to -O2 -g for debugging, I'd like this functionality to remain available.

Reply:
The reason I dropped the flags was because there were multiple -O0 and -g flags. They percolate up from the base config file(s). My intention was to bring the config in line with what the root does. I'll probably best to just drop those two lines altogether. The debug flags will percolate up anyway, so there's really no point (AFAIK).